lapack/unit_test/Test_Lapack_gesv.hpp

```diff
@@ -130,14 +130,16 @@ void impl_test_gesv(const char* mode, const char* padding, int N) {
 
   // Checking vs ref on CPU, this eps is about 10^-9
   typedef typename ats::mag_type mag_type;
-  const mag_type eps = 1.0e7 * ats::epsilon();
+  const mag_type eps = 2.0e7 * ats::epsilon();
   bool test_flag     = true;
   for (int i = 0; i < N; i++) {
     if (ats::abs(h_B(i) - h_X0(i)) > eps) {
       test_flag = false;
-      // printf( "    Error %d, pivot %c, padding %c: result( %.15lf ) !=
-      // solution( %.15lf ) at (%d)\n", N, mode[0], padding[0],
-      // ats::abs(h_B(i)), ats::abs(h_X0(i)), int(i) );
+      printf(
```

Comment:
Can this be guarded by a verbose flag or triggered if failure occurs? These tests have some noisy output (I've meant to open an issue but forgot)

Reply:
Yeah, this is an oversight of mine...